.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: Create an issue in tree-sitter-scala
+body:
+  - type: input
+    id: commit
+    attributes:
+      label: Commit of tree-sitter-scala you tested this on
+      description: |
+        Make sure you're validating you're on the latest commit of
+        tree-sitter-scala. Note that depending on where you're experiencing the
+        issue there is a chance you're on an old commit.
+    validations:
+      required: true
+
+  - type: textarea
+    id: code-sample
+    attributes:
+      label: A code sample showing the error
+      description: Try to minimize the code sample as much as possible
+      placeholder: |
+        ```scala
+          val y = x match {
+            case A(_) if a => 1
+          }
+        ```
+    validations:
+      required: true
+
+  - type: textarea
+    id: error
+    attributes:
+      label: Show the error node
+      description: Show us the output and where the ERROR node is appearing
```

Comment:
I think we need to put somewhere instructions on how to obtain this output

Reply:
Yea I was thinking about putting a section in the contributing docs about obtaining it. Let me create a section in there and add it to https://github.com/tree-sitter/tree-sitter-scala/pull/149 and then I'll link it from here.

Reply:
Alright, did that and updated it.